scripts/pre_commit_linter.py

```diff
@@ -1071,19 +1071,20 @@ def _lint_all_files(self):
         file_groups_to_lint = [
             html_files_to_lint_for_css, css_files_to_lint,
             js_files_to_lint, py_files_to_lint]
-        number_of_files_to_lint = sum(
-            len(file_group) for file_group in file_groups_to_lint)
+        # number_of_files_to_lint = sum(
```

Comment:
Please don't comment out code. Either delete it or keep it.

Reply:
Oh sorry for that. I just forget to remove after the debugging.